lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -223,20 +223,23 @@ class ReportRenderer {
     // }
 
     if (scoreHeader) {
-      const defaultGauges = [];
-      const customGauges = [];
+      // Group gauges in this order: mainstream, pwa, plugins.
+      const mainstreamGauges = [];
+      const customGauges = []; // PWA.
+      const pluginGauges = [];
       for (const category of report.reportCategories) {
         const renderer = specificCategoryRenderers[category.id] || categoryRenderer;
         const categoryGauge = renderer.renderScoreGauge(category, report.categoryGroups || {});
 
-        // Group gauges that aren't default at the end of the header
-        if (renderer.renderScoreGauge === categoryRenderer.renderScoreGauge) {
-          defaultGauges.push(categoryGauge);
+        if (Util.isPluginCategory(category.id)) {
+          pluginGauges.push(categoryGauge);
+        } else if (renderer.renderScoreGauge === categoryRenderer.renderScoreGauge) {
```

Comment:
this seems like an interesting way to tell, I guess the point is "do we render this gauge exactly like the others or does the gauge look different"?

Reply:
i 100% don't understand `renderer.renderScoreGauge === categoryRenderer.renderScoreGauge`.

Reply:
we weren't alone 😆 https://github.com/GoogleChrome/lighthouse/pull/6526/files#r234373117

Reply:
if it's confusing, should keep the comment :) (or make it more informative!)

Reply:
done